packages/desktop-client/src/components/common/Modal2.tsx

```diff
@@ -87,7 +88,10 @@ export const Modal = ({
     >
       <ReactAriaModal>
         {modalProps => (
-          <Dialog aria-label="Modal dialog">
+          <Dialog 
+            aria-label="Modal dialog"
+            className={`${css(styles.lightScrollbar)}`}
```

Comment:
The issue was because the styling includes the ```&``` combinator

Should be fixed now:

 ![image](https://github.com/user-attachments/assets/517767a1-c66b-42f6-a8f6-9b583b9682d4)